ecocodejava.properties

```diff
@@ -29,4 +29,10 @@ defaults.mavenArtifactId=ecocode-java-plugin
 1.4.3.sqVersions=[9.9,LATEST]
```

Comment:
This is the one that needs to be marked to end compatibility with 10.3. Sorry for the confusion.

Reply:
no problem. thank you.

Reply:
all is ok now.